### ISSUE TITLE
Nestjs: Failed to load bindings, pure JS will be used

### ISSUE THREAD

**Issue by padi-dev-thuyld** [BitGo/BitGoJS#4347]
## Environment Details

- **OS:** Mac
- **Node Version:** 18.13.0
- **Yarn Version:** 1.22.19
- **BitGoJS Version:** 37.1.2
- **BitGo Environment:** testnet
- **Nestjs:** 9.x

## Current Behavior

bigint: Failed to load bindings, pure JS will be used 
![Screen Shot 2024-03-14 at 11 53 50](https://github.com/padi-dev-thuyld/laravel-api-template/assets/80658278/eed5c75c-7d4c-44ae-b406-1001ca22383c)

**Comment by padi-dev-thuyld** [comment]
> Hi - I do not have permission to view that picture. Please specify the exact commands and environment you use so we can replicate the issue.

@davidkaplanbitgo updated.

**Comment by davidkaplanbitgo** [comment]
Closing this issue as this isn't an issue - we assume in the package that we will use the JS implementation of `bigint`

**Comment by Asd2812** [comment]
Re-opening this issue as I have had similar issues now. Currently working in the following environment:
OS: Mac
Node Version: 16.20.2
Yarn Version: 1.22.11
BitGoJS Version: 19.14.0

I am getting the exact same set of errors whereby the bigint package requires a JS implementation and there are duplicate polkadot files present. How did the error dissipate for you @padi-dev-thuyld ?

**Comment by padi-dev-thuyld** [comment]
> Adding a comment to this issue as I have had similar issues now. Currently working in the following environment: OS: Mac Node Version: 16.20.2 Yarn Version: 1.22.11 BitGoJS Version: 19.14.0
> 
> I am getting the exact same set of errors whereby the bigint package requires a JS implementation and there are duplicate polkadot files present. How did the error dissipate for you @padi-dev-thuyld ?

I have switched to using the API instead of using the package. @Asd2812

**Comment by Asd2812** [comment]
Yeah was trying to avoid needing to do that but I don't think I have any other options, thanks a lot @padi-dev-thuyld
